src/components/_channel/CollectorsBox/CollectorsBox.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react'
+
+import { Avatar } from '@/components/Avatar'
+import { AvatarGroup, AvatarGroupSingleAvatar } from '@/components/Avatar/AvatarGroup'
+import { ListItem } from '@/components/ListItem'
+import { Text } from '@/components/Text'
+import { SvgActionPlus } from '@/components/_icons'
+import { DialogModal } from '@/components/_overlays/DialogModal'
+import { cVar } from '@/styles'
+
+import { CollectorsBoxWrapper, PlusIconBackground, PlusIconWrapper } from './CollectorsBox.styles'
+
+type Collector = AvatarGroupSingleAvatar & { nftsAmount?: number }
+
+export type CollectorsBoxProps = {
+  collectors: Collector[]
+}
+
+const MAX_SHOWED_COLLECTORS = 5
```

Comment:
Shouldn't this be a prop instead?

Reply:
hmm, good idea, but I will keep 5 as a default value